appengine/pond/duck/js/datastore.js

```diff
@@ -107,7 +107,23 @@ Pond.Duck.Datastore.getDuck = function(duckKey, onSuccess) {
  *    successfully.
  */
 Pond.Duck.Datastore.getMatchRequest = function(onSuccess) {
-    BlocklyStorage['makeRequest']('pond-storage/rank', '', onSuccess, null, 'GET');
+    BlocklyStorage['makeRequest']('pond-storage/rank', '', onSuccess);
```

Comment:
Should this be a GET instead of a POST?

Reply:
I changed it to POST because it doesn't just request data, it also creates and stores a MatchRequest.